src/config.ts

```diff
@@ -137,6 +142,22 @@ export interface Config {
    */
   samplingRate?: number;
 
+  /**
+   * Specifies how to use incoming trace context headers. The following options
+   * are available:
+   *   'default' -- Trace context headers will be respected for incoming
+   *   requests that support it. A new trace will be created for requests
+   *   without trace context headers.
+   *   'require' -- Same as default, but traces won't be created for requests
+   *   without trace context headers. This should not be set for end user-facing
```

Comment:
Please also mention that the option should only be set if the headers are in the trustable. If an attacker can control the heade, they may cause excessive tracing.

Reply:
`require` is more strict than `default`, so I don't think that applies.